pootle/templates/browser/_table.html

```diff
@@ -42,7 +42,11 @@
 
       {% if 'total' in table.fields %}
       <td id="total-words-{{ item.code|cssid }}" class="stats-number total not-inited">
-        <a class="non-zero" href="{{ item.href_all }}"></a>
+	{% if can_translate %}
```

Comment:
Please do not use tabs for indenting, all our HTML files contain 2-space indents.